main.go

```diff
@@ -2804,6 +2810,61 @@ The following options are recognized:
 		return nil
 	})
 
+	r.Dispatch("uuid", &Help{
+		Summary: "Generate a version 4 uuid value and write it to the specified path and/or key. If no key is provide the key `uuid` is used",
+		Usage:   "safe uuid PATH[:KEY]",
+		Type:    NonDestructiveCommand,
+		Description: ``,
+	}, func(command string, args ...string) error {
+		rc.Apply(opt.UseTarget)
+
+		if len(args) == 0 {
+			r.ExitWithUsage("uuid")
+		}
+
+		u := uuid.NewRandom()
+
+		stringuuid := u.String()
+
+		v := connect(true)
+
+		for len(args) > 0 {
+			var path, key string
+			if vault.PathHasKey(args[0]) {
+				path, key, _ = vault.ParsePath(args[0])
+				args = args[1:]
```

Comment:
if the loop goes away, so does this shift.

Reply:
Just to confirm. Your saying 2835 should go away as args is no longer going to be used.

Reply:
correct